frame/executive/src/lib.rs

```diff
@@ -272,9 +294,11 @@ where
 		uxt: Block::Extrinsic,
 		encoded_len: usize,
 		to_note: Option<Vec<u8>>,
+		check_signature: bool,
 	) -> ApplyExtrinsicResult {
 		// Verify that the signature is good.
-		let xt = uxt.check(&Default::default())?;
+		let xt = if check_signature { uxt.check(&Default::default())? }
```

Comment:
Why not pass extra parameters to `Checkable` instead? An extra trait `UnsafeConvert` seems to complicate everything more than help imho.

Reply:
Yeah, I thought the same, but I'm not 100% sure.

Reply:
Well it was my first lazy thought, but because it is called "Checkable", it is supposed to check, not to skip check :)

Reply:
Yeah that would be my mental model as well, however a `skip_check` parameter would a little bit cleaner.